Gemfile

```diff
@@ -27,7 +27,7 @@ group(:development, :test) do
   gem 'packaging', *location_for(ENV['PACKAGING_LOCATION'] || '~> 0.99')
   gem 'rake', require: false
   gem 'rspec', '~> 3.0', require: false
-  gem 'rubocop', "~> 0.52.1", require: false
+  gem 'rubocop', "~> 0.75", require: false
```

Comment:
Do we want to go from three digits to two here? I don't know what kinds of changes go into their Y releases, but seeing as there's no 1.0 yet, we might wanna be more careful? 
Or this could help us stay more up-to-date with rubocop, if we make changes as we go..
Things to think about..

Reply:
looking at https://github.com/rubocop-hq/rubocop/releases it seems like they rarely go beyond a .1 so it's probably ok.